fxgl-core/src/main/kotlin/com/almasb/fxgl/core/math/PerlinNoiseGenerator.kt

```diff
@@ -23,14 +23,17 @@ internal object PerlinNoiseGenerator {
     private val gx = FloatArray(NOISE_TABLE_SIZE)
     private val gy = FloatArray(NOISE_TABLE_SIZE)
 
+    private val seed = inputSeed;
```

Comment:
I don't think we need `val seed` here, we might be able to just use `inputSeed` directly